x-pack/plugins/maps/public/classes/layers/blended_vector_layer/blended_vector_layer.test.tsx

```diff
@@ -86,6 +86,7 @@ describe('getSource', () => {
         geoField: sourceDescriptor.geoField,
         applyGlobalQuery: sourceDescriptor.applyGlobalQuery,
         applyGlobalTime: sourceDescriptor.applyGlobalTime,
+        applyForceRefresh: sourceDescriptor.applyForceRefresh,
```

Comment:
See comment above this line
```
// Purposely grabbing properties instead of using spread operator
// to ensure type check will fail when new properties are added to AbstractESSourceDescriptor.
// In the event of type check failure, ensure test is updated with new property and that new property
// is correctly passed to clustered source descriptor.
```

haha - it worked